speechbrain/lobes/models/HifiGAN.py

```diff
@@ -1145,14 +1406,19 @@ def __init__(
         self.feat_match_loss_weight = feat_match_loss_weight
         self.l1_spec_loss = l1_spec_loss
         self.l1_spec_loss_weight = l1_spec_loss_weight
+        self.mseg_dur_loss = mseg_dur_loss
+        self.mseg_dur_loss_weight = mseg_dur_loss_weight
 
     def forward(
         self,
+        stage,
```

Comment:
Hi @Chaanks, adding this argument may cause a fail for the normal hifigan recipe, please run the hifigan recipe to make sure that everything works before merge, thanks.

Reply:
Thank you, @BenoitWang, for pointing this. I also made a minor modification by changing the location of 'ljspeech_prepare' to avoid code duplication. Yes, I will do it. Can you double-check it with me @mhn226 ?